lambda/session/lambda_functions.py

```diff
@@ -78,6 +86,8 @@ def _delete_user_session(session_id: str, user_id: str) -> Dict[str, bool]:
     deleted = False
     try:
         table.delete_item(Key={"sessionId": session_id, "userId": user_id})
+        bucket = s3_resource.Bucket(s3_bucket_name)
+        bucket.objects.filter(Prefix=f"images/{session_id}").delete()
```

Comment:
We certainly don't do this everywhere (anywhere?) other than maybe prompt templates to support versioning, but should we consider something like this being configurable and having logical deletes? At some point we'll want auditability most likely and I'm not sure how likely we'll want to delete stuff like this? Probably a bigger discussion as it would affect more than just this.